### PR TITLE
Various fixes to mcpi modded

### PR DESCRIPTION
**Pull request by mobilegmYT**
Some other mcpi changes are coming soon so don't merge for now

### CODE REVIEW

**Comment by mobilegmYT** [comment]
Not tested

File: apps/Minecraft Pi (Modded)/install [hunk 1]
```diff
@@ -25,15 +29,15 @@ wget -qO- https://raw.githubusercontent.com/MCPI-Revival/mcpi-packages/master/in
 # Choose launcher to use
 PS3="Which launcher for MCPI Modded would you like to use? Need help deciding? Ask here: https://discord.com/invite/aDqejQGMMy. You can also go to the github links for the respective launchers to see how they look.
 "
-options=("gMCPIL (recommended, written in Vala and C) (https://github.com/MCPI-Revival/gMCPIL)" "jMCPIL (better looking GUI but requires Java) (https://github.com/MCPI-Revival/jMCPIL)")
+options=("jMCPIL (reccomended, better looking GUI but requires Java) (https://github.com/MCPI-Revival/jMCPIL)" "gMCPIL (written in Vala and C) (https://github.com/MCPI-Revival/gMCPIL)")
```
Comment (Jai-JAP) [inline apps/Minecraft Pi (Modded)/install:32]: its recommended not reccomended
Fix this typo

File: apps/Minecraft Pi (Modded)/install [hunk 2]
```diff
@@ -1,37 +1,66 @@
...
 function error {
   echo -e "\\e[91m$1\\e[39m"
   exit 1
 }
 
+# Wait for apt locks to be opened
 "${DIRECTORY}/api" apt_lock_wait
 
+# Remove old depricated packagecloud repo
+if test -f "/etc/apt/sources.list.d/Alvarito050506_mcpi-devs.list"; then
```
Comment (Itai-Nelken) [inline apps/Minecraft Pi (Modded)/install:15]:
```suggestion
if [ -f "/etc/apt/sources.list.d/Alvarito050506_mcpi-devs.list" ]; then
```
So it's consistent with the rest of the svripts.

File: apps/Minecraft Pi (Modded)/install [hunk 3]
```diff
@@ -1,37 +1,66 @@
...
+
 # Remove old MCPILs
 sudo apt-get remove -y mcpil-r mcpil &>/dev/null || true
 
+# Install mcpi-packages apt repo
 wget -qO- https://raw.githubusercontent.com/MCPI-Revival/mcpi-packages/master/install.sh | grep -v 'sudo apt update' | sudo bash - || error "Failed to install APT repo!"
 
-# Choose MCPIL to use
-PS3="Which launcher for MCPI Modded would you like to use? Need help deciding? Ask here: https://discord.com/invite/aDqejQGMMy
+# Choose launcher to use
```
Comment (Itai-Nelken) [inline apps/Minecraft Pi (Modded)/install:29]:
```suggestion
# Choose which launcher to use
```

File: apps/Minecraft Pi (Modded)/install [hunk 4]
```diff
@@ -1,37 +1,66 @@
...
+esac
+fi
+
+# Optionally install Python API
+read -p "Do you wish to install the MCPI API for the ability to run Python API scripts? (select yes unless you know what your doing!)" choice
+case "$choice" in 
+  y|Y ) "${DIRECTORY}/pkg-install" "python3-pip" "$(dirname "$0")" || error "Failed to install python3-pip which is needed to install the API!" && pip3 install mcpi || error "The pip3 command failed to install the MCPI API!";;
+  n|N ) echo "Not installing API... proceeding";;
+  * ) echo "Invalid option! Please type 'y' or 'n'";;
+esac
```
Comment (Itai-Nelken) [inline apps/Minecraft Pi (Modded)/install:66]: Put this whole read and case inside a loop so if user enters an invalid answer, they can enter an answer again.

Comment (mobilegmYT) [reply apps/Minecraft Pi (Modded)/install:66]: > 

Like a forever loop and use `break`?

Comment (Itai-Nelken) [reply apps/Minecraft Pi (Modded)/install:66]: yes

**Comment by TheBrokenRail** [comment]
This can probably be closed.

**Comment by mobilegmYT** [comment]
😭 why must you do this to me tbr

**Pull request closed without merging**